test-requirements.txt

```diff
@@ -1 +1,3 @@
 odoo_test_helper
+
+odoo14-addon-attribute-set-completeness @ git+https://github.com/acsone/odoo-pim.git@14-attribute_set_completeness-xbo#subdirectory=setup/attribute_set_completeness
```

Comment:
Please mention that it depends on https://github.com/OCA/odoo-pim/pull/86.

Also remember to remove this before merge pls

Reply:
added to description